packages/discord.js/src/util/Constants.js

```diff
@@ -376,61 +376,6 @@ exports.InviteScopes = [
  */
 exports.IntegrationExpireBehaviors = createEnum(['REMOVE_ROLE', 'KICK']);
 
-/**
- * The type of a message, e.g. `DEFAULT`. Here are the available types:
- * * DEFAULT
- * * RECIPIENT_ADD
- * * RECIPIENT_REMOVE
- * * CALL
- * * CHANNEL_NAME_CHANGE
- * * CHANNEL_ICON_CHANGE
- * * CHANNEL_PINNED_MESSAGE
- * * GUILD_MEMBER_JOIN
- * * USER_PREMIUM_GUILD_SUBSCRIPTION
- * * USER_PREMIUM_GUILD_SUBSCRIPTION_TIER_1
- * * USER_PREMIUM_GUILD_SUBSCRIPTION_TIER_2
- * * USER_PREMIUM_GUILD_SUBSCRIPTION_TIER_3
- * * CHANNEL_FOLLOW_ADD
- * * GUILD_DISCOVERY_DISQUALIFIED
- * * GUILD_DISCOVERY_REQUALIFIED
- * * GUILD_DISCOVERY_GRACE_PERIOD_INITIAL_WARNING
- * * GUILD_DISCOVERY_GRACE_PERIOD_FINAL_WARNING
- * * THREAD_CREATED
- * * REPLY
- * * CHAT_INPUT_COMMAND
- * * THREAD_STARTER_MESSAGE
- * * GUILD_INVITE_REMINDER
- * * CONTEXT_MENU_COMMAND
- * @typedef {string} MessageType
```

Comment:
this is probably referenced elsewhere in the docs. What will those point to now?

Reply:
As of right now? Nowhere. But that was the case with all the other "enums" that were removed on the original pr.

I assume there will be changes to the website itself to link to the discord-api-types enums or something

Reply:
They are at least here https://discord.js.org/#/docs/discord.js/main/class/Message?scrollTo=type
But if that's the case I'd love to see that soon!

Reply:
Yeah, because that constant in particular wasn't removed on the other pull request, despite it not being referenced anywhere else (most likely an oversight)

All the other ones do not link to anything on the website. i.e. https://discord.js.org/#/docs/discord.js/main/class/Activity?scrollTo=type